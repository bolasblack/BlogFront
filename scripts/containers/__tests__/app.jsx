import React from 'react'
import App from '../app'

describe('App', () => {
  let React

  beforeEach(() => {
    jest.resetModules()
    React = require('react')
  })

  it('successfully rendered', () => {
    const renderer = require('react-test-renderer')
    /* const elem = renderer.create(<App.App />)*/
    /* const tree = elem.toJSON()*/
    /* expect(tree).toMatchSnapshot()*/
    pending('fill')
  })

  it('can also response some event', () => {
    const {shallow} = require('enzyme')
    /* const elem = shallow(<App.App />)*/
    pending('fill')
  })
})
