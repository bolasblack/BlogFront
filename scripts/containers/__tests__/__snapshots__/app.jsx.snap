exports[`App successfully rendered 1`] = `
<div
  className="app-page"
  onClick={undefined}>
  It works
</div>
`;
